crates/eigen-cli/src/lib.rs

```diff
@@ -123,7 +210,7 @@ pub fn execute_command(command: Commands) -> Result<(), EigenCliError> {
 #[cfg(test)]
```

Comment:
probably better to move tests to their own files, but later on is fine.

Reply:
Actually , we are following same file same test convention in the whole sdk